src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -833,9 +833,9 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 #endif
 	}
 	case SN_Create: {
-		if (!is_element_type_primitive (fsig->ret))
```

Comment:
That can be done once instead for every different switch case.

Reply:
Originally, I was planing to do that. As I implement it, I found that there are all kinds of scenarios. Some methods has both vector inputs and outputs, some methods has only vector inputs, some methods has only vector outputs. Had I do it in a different function, it would involve switch-case as well.